tests/integration/targets/helm_kubeconfig/tasks/main.yml

```diff
@@ -1,19 +1,21 @@
 ---
 - name: Test helm with in-memory kubeconfig
   include_tasks: "from_in_memory_kubeconfig.yml"
+  loop_control:
+    loop_var: test_helm_version
+  with_items:
+    - "v3.10.3"
```

Comment:
Is this change intended in this PR?

Reply:
Yes it is. The reference is here https://github.com/ansible-collections/kubernetes.core/pull/670#issuecomment-1947945547